client/config/default.json

```diff
@@ -6,5 +6,16 @@
     "geocode": {
         "mapsApiKey": "",
         "autocompleteDelay": 2000
+    },
+    "analytics": {
+        "ga": {
+            "enabled": false,
+            "ua": ""
+        },
+        "hotjar": {
+            "enabled": false,
```

Comment:
what are those? Maybe you can name them with clearer names....

Reply:
i assume you mean hjid and hjsv - these are the names of variables the library takes (https://github.com/abdalla/react-hotjar#readme) and also the names of variables on hotjar's tracking code info (https://help.hotjar.com/hc/en-us/articles/115011639927-What-is-the-Hotjar-Tracking-Code-), so these seem to be standard.
hjid stands for hotjar id and hjsv is hotjar service version.

Reply:
Yes I get they're hotjar conventions, however as we require them on initializaing hotjar , and they're not just loaded to `process.env` just wondered if they could be in our config more clear.